action_server/tests/action_server_tests/test_cli/test_help.txt

```diff
@@ -15,6 +15,8 @@ positional arguments:
     package             Utilities to manage the action package
     env                 Commands related to the environment handling
     cloud               Utilities to perform Control Room operations
+    get-user-oauth-config-path
```

Comment:
I think the command structure could have an `oauth2`  with subcommands (I think it may grow, so, preparing for that...).

So, it can be something as:

`action-server oauth2 user-config-path`

Note all references should be to `oauth2` instead of `oauth`.

Reply:
@fabioz done!